tss-esapi/src/structures/buffers/public.rs

```diff
@@ -233,7 +233,7 @@ impl PublicBuilder {
                         Error::local_error(WrapperErrorKind::ParamsMissing)
                     })?,
                     unique: self.rsa_unique_identifier.ok_or_else(|| {
-                        error!("RSA unique identifier has not been set in the PublicBuilder even though the RSA algorithm had been selected.");
+                        error!("RSA unique identifier has not been set in the PublicBuilder even though the RSA algorithm had been selected. Consider using: .with_rsa_unique_identifier(&Default::default())");
```

Comment:
Could you please replace the `Default` with the corresponding type, e.g. `PublicKeyRsa` in this case? Seemsa tiny bit easier for people to figure out what they're doing, and what else they could be doing.

Reply:
Fixed!